src/formFrom.js

```diff
@@ -39,25 +40,26 @@ export default formFields => {
       fetch(this.props.endpoint, {
         method: 'POST',
         headers: {
+          ...auth.header(),
```

Comment:
Does this includes the header on `sign-up` too? which is why I was seeing [this](https://github.com/yujinjcho/rcruitme/pull/38#issuecomment-455291418)?

Reply:
Yeah, includes for `sign-up` post too, but shouldn't result in the error you were seeing..

So actually I realized this branch doesn't include [these changes](https://github.com/yujinjcho/rcruitme-front/pull/17), and I had just pulled them in locally to test. If I remove them and go strictly off this branch, sign-up doesn't work for me either, but for a different reason (missing `redirect` param). Are you sure that's not actually why failing for you? The other error you're seeing might be left over from before you cleared cookies?